.github/workflows/run-forge.yaml

```diff
@@ -23,8 +23,18 @@ on:
       FORGE_RUNNER_DURATION_SECS:
         required: false
         type: string
-        default: 480
+        default: 7200
         description: Duration of the forge test run
+      FORGE_ENABLE_HAPROXY:
+        required: false
+        type: string
+        default: false
+        description: Is HAProxy enabled
+      FORGE_NUM_VALIDATORS:
+        required: false
+        type: string
+        default: 100
```

Comment:
can we remove the default, such that the number of validators is equal to the number set in the individual test cases?